src/Workspaces/Core/Portable/Telemetry/TelemetryLogging.cs

```diff
@@ -18,7 +18,7 @@ internal static class TelemetryLogging
         public const string KeyName = "Name";
         public const string KeyValue = "Value";
         public const string KeyLanguageName = "LanguageName";
-        public const string KeyMetricName = "MetricName>";
+        public const string KeyMetricName = "MetricName";
```

Comment:
>MetricName [](http://example.com/codeflow?start=45&length=10)

No functional change here, just noticed this oddity while debugging. This name is never sent over as part of the telemetry.